ci-wheel.Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN case "${LINUX_VER}" in \
         echo 'APT::Update::Error-Mode "any";' > /etc/apt/apt.conf.d/warnings-as-errors \
         && apt update -y \
         && apt install -y \
-          jq build-essential software-properties-common which wget gcc zlib1g-dev libbz2-dev \
+          debianutils build-essential software-properties-common \
```

Comment:
Need to use [`debianutils` to get `which` on Ubuntu]( https://packages.ubuntu.com/focal/amd64/debianutils/filelist )